sdk/storage/azure-storage-files-datalake/vcpkg/vcpkg.json

```diff
@@ -12,7 +12,7 @@
   "license": "MIT",
   "dependencies": [
     {
-      "name": "azure-storage-common-cpp",
+      "name": "azure-storage-blobs-cpp",
```

Comment:
What motivated this change @antkmsft ?

@Jinming-Hu is this dependency hierarchy correct?

Reply:
Yes